src/porepy/models/geometry.py

```diff
@@ -141,8 +141,7 @@ def wrap_grid_attribute(
         attr: str,
         *,
         dim: int,
-        inverse: bool = False,
-    ) -> pp.ad.SparseArray:
+    ) -> pp.ad.DenseArray:
```

Comment:
This is a change that I am unsure about. If small and sparse grid attributes are needed in production, this is a good choice. Could you please elaborate on this choice?